### PR TITLE
New changes

### PR DESCRIPTION
**Pull request by vidban**
Added an interact option in marshmallow.md

### CODE REVIEW

**Comment by vidban** [comment]
Added a new link "interact" in the marshmallow.md with two further storylines

**Comment by vidban** [comment]
Please review this, @casey-collab

File: english/interact/interact.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+The marshmallow hand sinks further into the wall behind him.
+
+You close your eyes and contemplate your next step.
+
+When you open your eyes, the marshmallow hand is nowhere to be seen....
+
+You then:
+
+[Take your lighter out] (lighter/lighter.md)
+[Bend down on one knee] (bend/bend.md)
```
Comment (casey-collab) [inline english/interact/interact.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by vidban** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :sunny:

**Pull request merged**